### PR TITLE
IPFS Endeavours

### PR DESCRIPTION
**Pull request by RichardLitt**
This is the transcript and video for the IPFS Endeavors video that Juan gave in October at Protocol Labs.

### CODE REVIEW

**Comment by RichardLitt** [comment]
@lgierth Can we publish this?

File: src/26-ipfs-endeavors/index.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+# this id is part of the url, and should only contain:
+#   letters, numbers, dashes.
+id: 26-ipfs-endeavors
+
+breadcrumbs:
+  - {name: "26-ipfs-endeavors", link: "./" }
+
+# the date here should be set to the final publication date,
+# on the day it is published.
+date: 2015-05-05
```
Comment (VictorBjelkholm) [inline src/26-ipfs-endeavors/index.md]: Guess this would be changed after merged and before published?

Comment (RichardLitt) [inline src/26-ipfs-endeavors/index.md]: Yes!

File: src/26-ipfs-endeavors/index.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+# this is the name of the main author(s)
+author: Juan Benet
+
+# technical details required for the software, don't change these.
+baseurl: ..
+template: tmpl/layouts/post.html
+collection: posts
+---
+
+In October, at Protocol Labs, Juan Benet sat down to do a quick overview of the main IPFS endeavors. This video runs through the different projects; a transcript is provided below.
```
Comment (jbenet) [inline src/26-ipfs-endeavors/index.md]: minor comment: this was part of IPFS calls, not protocol labs. the "at protocol labs" "making it a place" thing is just also misleading in isolation here.

Comment (jbenet) [inline src/26-ipfs-endeavors/index.md]: just remove the "at Protocol Labs", it's not even relevant.

**Comment by jbenet** [comment]
minor comment, thats it

File: src/26-ipfs-endeavors/index.md (hunk 2 above)
Comment (RichardLitt) [inline src/26-ipfs-endeavors/index.md]: Done!

**Comment by jbenet** [comment]
ok thanks

**Comment by diasdavid** [comment]
_Closing ancient posts that never got posted._

**Pull request closed without merging**